src/templates/menu.twig

```diff
@@ -5,108 +5,132 @@
+                <li><div class="d-flex justify-content-between"><span>Cuba Libre</span> <span>9.9</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Mojito</span> <span>9.9</span></div></li>
+            </ul>
+        </div>
+        <div class="col-sm">
+            <h3>Spirits</h3>
+            <ul class="list-unstyled">
+                <li><div class="d-flex justify-content-between"><span>Metaxa 12*</span> <span>7.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Tullamore Dew</span> <span>6.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Jameson Whiskey</span> <span>6.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Bombay, Añejo Suave</span> <span> 6.8</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Ron Cubay, Añejo Suave</span> <span> 6.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Tequila, El Jimador, Reposado</span> <span>6.2</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Vodka Finlandia</span> <span>5.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Bailey's</span> <span>5.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Jagermeister</span> <span>5.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Graham's Tawny Porto</span></div></li>
+            </ul>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm">
+            <h2>Something to Eat</h2>
+            <ul class="list-unstyled">
+                <li><div class="d-flex justify-content-between"><span>Soup of the day</span> <span>5.8</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Chipotle Citrus Chicken Sandwich</span> <span>5.4</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Avocado Toast</span> <span>5.6</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Super BLT Sandwich</span> <span>12.8</span></div></li>
+            </ul>
+        </div>
+        <div class="col-sm">
+            <h2>Sweets</h2>
+            <ul class="list-unstyled">
+                <li><div class="d-flex justify-content-between"><span>Homemade cakes of the day</span> <span>5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Key Lime and Coconut</span> <span>1.5</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Ice cream</span> <span>2</span></div></li>
+                <li><div class="d-flex justify-content-between"><span>Tiramisu</span> <span>6</span></div></li>
+            </ul>
+        </div>
+    </div>
```

Comment:
try adding sections for main parts of the menu (and corresponding headings if necessary)